File: Topics/Projetos-Estudo/fullcommerce-1/fullcommerce/website/src/js/utils/logger.js
function info(message) {
    console.log(`[INFO] - ${message}`);
}

function warning(message) {
    console.log(`[WARNING] - ${message}`);
}

function error(message) {
    console.log(`[ERROR] - ${message}`);
}



export default {
    info,
    warning,
    error
};

File: topics/Python/projetos/02-integration-pokemon/v2.0/dev/integration/utils/logger.py
from datetime import datetime
from pytz import timezone

from integration.constants import TIMEZONE_DEFAULT



def info(message):
    nowDatetime = datetime.now(timezone(TIMEZONE_DEFAULT)).strftime('%Y-%m-%d %H:%M:%S')
    print(f"[{str(nowDatetime)}]  [INFO]  -  {message}")


def error(message):
    nowDatetime = datetime.now(timezone(TIMEZONE_DEFAULT)).strftime('%Y-%m-%d %H:%M:%S')
    print(f"[{str(nowDatetime)}]  [ERROR]  -  {message}")
